### DISCUSSION TITLE
[Feature Request] Add zip compression for pass

### DISCUSSION THREAD

**Discussion by salos1982** [alexandercerutti/passkit-generator#128] [Ideas]
Hello, Currently pkpass that is generated have no compression. It would be better if it has compression.

**Comment by alexandercerutti** [comment]
Hi @salos1982, thanks for using `passkit-generator`.

I'm aware of the lack of compression in `passkit-generator`.
There's a precise reason for which I've chosen to not integrate it, and it is documented here, along with a suggested way that will let you overcome this limit:

- https://github.com/alexandercerutti/passkit-generator/wiki/Migrating-from-v2-to-v3#generate
- https://github.com/alexandercerutti/passkit-generator/wiki/API-Documentation-Reference#getasraw

Let me know if you have any doubts and how I can help you.

**Reply by salos1982** [reply]
Thanks for reply
